scripts/zones/Lower_Jeuno/npcs/Chululu.lua

```diff
@@ -43,16 +52,30 @@ function onTrigger(player,npc)
         player:startEvent(49); -- During quest "Rubbish Day"
     elseif (RubbishDay == QUEST_ACCEPTED and player:getCharVar("RubbishDayVar") == 1) then
         player:startEvent(197); -- Finish quest "Rubbish Day"
+		
+		
+    elseif (player:getFameLevel(JEUNO) >= 4 and CollectTarutCards == QUEST_COMPLETED and AllInTheCards == QUEST_AVAILABLE) then
+        player:startEvent(10110); -- Start quest "All in the Cards" with option
+	elseif (AllInTheCards == QUEST_ACCEPTED and cdate >= os.time()) then
+		player:startEvent(10111); -- During quest "All in the Cards" and same AllInTheCards_date value		
+	elseif (AllInTheCards == QUEST_ACCEPTED and cdate ~= os.time()) then
+		player:startEvent(10112); -- During quest "All in the Cards"  THIS ONE GIVES ANOTHER BATCH
+	elseif (CollectTarutCards == QUEST_COMPLETED and AllInTheCards == QUEST_COMPLETED) then
+		player:startEvent(10113); -- Start quest "All in the Cards"	repeat with option		
```

Comment:
This will need to check that AllInTheCards >= QUEST_ACCEPTED for people repeating the quest. Something like:
```lua
elseif AllInTheCards >= QUEST_ACCEPTED then
    if cdate >= os.time() then
        player:startEvent(10111) 
    elseif cdate == 0 then
        player:startEvent(10113)
    elseif cdate < os.time() then
        player:startEvent(10112) 
    end
```
Once a quest is marked as complete it stays that way unless it's manually deleted. Adding the quest again doesn't set it back to accepted. Once you complete the quest, it will always give you cs 10113 and never continue to the stuff past this block. I'm not sure how this quest is supposed to progress, but you will probably need to add a localvar to track whether the player has denied this quest/timer isn't up yet so that they can talk again and get the next quest dialogue.